docs/reference/members.md

```diff
@@ -58,3 +78,25 @@ projects_and_groups:
       enforce: true
       keep_bots: false
 ```
+
+### Custom Roles (GitLab Ultimate Only)
+Assigning of Custom Roles to `group_members` is supported within GitLabForm configuration.
+
+!!! warning
+
+    * user `access_level` MUST still be supplied and MUST match the base_access_level of the custom role
+
+```yaml
+projects_and_groups:
+group_1/*:
+    group_members:
+      groups:
+        another-group:
+          group_access: no access
+      users:
+        my-user:
+          access_level: owner
+          member_role_id: 2
```

Comment:
In this example, shouldn't this be under the `another_group`?

Reply:
@amimas -> from reading https://docs.gitlab.com/ee/api/members.html#add-a-member-to-a-group-or-project, member_role_id is only support for a per user basis. I'll have to test and see if you can set a whole group to have a member_role_id